platformio.ini

```diff
@@ -505,20 +505,12 @@ extends  = common_avr8
 board    = reprap_rambo
 
 #
-# FYSETC F6 V1.3
+# FYSETC F6 V1.3 / V1.4
 #
-[env:FYSETC_F6_13]
+[env:FYSETC_F6]
 platform = atmelavr
 extends  = common_avr8
-board    = fysetc_f6_13
-
-#
-# FYSETC F6 V1.4
-#
-[env:FYSETC_F6_14]
-platform = atmelavr
-extends  = common_avr8
-board    = fysetc_f6_14
+board    = fysetc_f6
```

Comment:
We could probably use `fysetc_f6_13` and eliminate our own custom board JSON file.

Yeah, the comment in the JSON file indicates the wrong version, but does that actually show up anywhere? I suspect it does not matter.